modules/specialized-guides/pages/salt/salt-monitoring.adoc

```diff
@@ -0,0 +1,36 @@
+[[salt-monitoring]]
+
+= Monitoring salt events
+
+In some cases {salt} event bus could publish significant amount of events with the high rate, what could make it hard to analize.
+Getting the statistics based on the events published in the {salt} event bus can help to identify possible weak points and perform {salt} internals profiling.
+
+[[saline]]
+= Saline
+
```

Comment:
I'm wondering whether it is needed to say here at the beginning that you run commands such as mgradm or mgrctl on the "{productname} container host"?

Reply:
Yes, good point, will add it and will also put some examples of the output of Grafana